manifests/fingerprinting-protection-debugger.yml

```diff
@@ -0,0 +1,13 @@
+---
+description: Fingerprinting Protection Debugger Addon
+repo-prefix: fingerprinting-protection-debugger
+active: true
+private-repo: false
+additional-emails: ["fkilic@mozilla.com"]
```

Comment:
Please add a group email if possible, otherwise more individuals should probably be listed.

Reply:
I just added Frederik Braun, Tom Ritter, and Tim Huang to the list. Thank you!